app/controllers/api/realtime/sessions_controller.rb

```diff
@@ -37,7 +37,8 @@ def index
         data[:time_from] = Time.strptime(data[:time_from].to_s, '%s')
         data[:time_to] = Time.strptime(data[:time_to].to_s, '%s')
 
-        respond_with FixedSession.filtered_json(data, data[:limit], data[:offset])
+        respond_with sessions: FixedSession.filtered_json(data, data[:limit], data[:offset]),
+                     availableSessionsCount: MobileSession.filter(data).count
```

Comment:
I'd consider calling this simply `amount` or `count` or `quantity` or `total` in this context it should be clear, we're hitting the /sessions endpoint and getting back `{ sessions: ..., total: ... }`. but `{ sessions: ..., sessionAmount: ... }` is fine too I guess if you want to be very clear.

Reply:
_* I find that confusing. If it says `filterUtils` - that sounds like there is one filter that has many utils. If it says `filtersUtils` - that sounds like many filters that can have multiple utils. That's how I've been always using it. Is that really grammatically incorrect? It seems like it's needed ssses, cause it's changing the meaning._

Reply:
I agree that `FiltersUtils` sounds overkill, and I would also use it without the additional `s`, but in case of counting, I think I'm alright with that, and I use the form `sessionsCount`. This is because most of the time I know there are multiple sessions or whatever, and the name reflects that. It might also be in the context of an object that only deals with multiple instances of something. As an aside, I see a method like `getChildrenCount` in Android SDK, so I was wondering if that's also grammatically incorrect in this context, or not?

Reply:
>  Is that really grammatically incorrect? It seems like it's needed ssses, cause it's changing the meaning.
Yes, and when I see it I always imagine it in an old German professor's voice ;) "Filter utils" means utils for filters, utils related to filters. It's not a possessive adjective, so it doesn't need `'s` or `s'` depending on sing/plur. It doesn't really matter if there's one or more filters. Like "crocodile tears" and not "crocodile's tears" or "crocodiles' tears".

> As an aside, I see a method like getChildrenCount in Android SDK, so I was wondering if that's also grammatically incorrect in this context, or not?
Imo `getChildrenCount` and `getChildCount` are both correct and ok. what would be incorrect is `childrensCount` or `childsCount`. Besides that, I agree with your take on "count".